src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,72 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String WORDS_DIVIDER = ",";
+    private static final String SUPPLY_OPERATION = "supply";
+    private static final String BUY_OPERATION = "buy";
+    private static final String RESULT_OPERATION = "result";
+    private static final int FIRST_ELEMENT_INDEX = 0;
+    private static final int SECOND_ELEMENT_INDEX = 1;
+    private static final String EMPTY_STRING = "";
 
     public void getStatistic(String fromFileName, String toFileName) {
+        String data = readInputFile(fromFileName);
+        String report = EMPTY_STRING;
+        if (!data.isEmpty()) {
+            report += calculateStatistic(data);
+        }
+        if (!report.isEmpty()) {
+            writeReportToFile(report, toFileName);
+        }
```

Comment:
you don't need to check for a value presence

Reply:
done